src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -56,9 +56,12 @@ public void CheckProguardMappingFileExists ()
 			};
 			proj.SetProperty (proj.ReleaseProperties, KnownProperties.AndroidDexTool, "d8");
 			proj.SetProperty (proj.ReleaseProperties, KnownProperties.AndroidLinkTool, "r8");
+
 			using (var b = CreateApkBuilder ()) {
-				Assert.IsTrue (b.Build (proj), "build should have succeeded.");
 				string mappingFile = Path.Combine (Root, b.ProjectDirectory, proj.OutputPath, "mapping.txt");
+				// Projects must provide a $(AndroidProguardMappingFile) value to opt in
+				proj.SetProperty (proj.ReleaseProperties, "AndroidProguardMappingFile", mappingFile);
```

Comment:
I think we should make the `.csproj` in this test use `<AndroidProguardMappingFile>$(OutputPath)mapping.txt</AndroidProguardMappingFile>`, and then make sure we have this example in the release notes:

https://github.com/xamarin/xamarin-android/blob/55faee0fe9e489bc03cc8ac08140169ddac4c5b5/Documentation/release-notes/5304.md

Reply:
Good call, updated